src/org/traccar/database/NotificationManager.java

```diff
@@ -78,8 +78,10 @@ public void updateEvent(Event event, Position position) {
             usersToForward = new HashSet<>();
         }
         for (long userId : users) {
-            if (event.getGeofenceId() == 0 || Context.getGeofenceManager() != null
-                    && Context.getGeofenceManager().checkItemPermission(userId, event.getGeofenceId())) {
+            if ((event.getGeofenceId() == 0 || Context.getGeofenceManager() != null
+                    && Context.getGeofenceManager().checkItemPermission(userId, event.getGeofenceId()))
+                    && (event.getMaintenanceId() == 0
```

Comment:
Do we need to check if manager is null? If not, maybe we should remove geofence manager check?